<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="TachometerWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="BatteryWidget" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="SevenSegmentClock" name="widget_3" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="maximum">
       <number>270</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TachometerWidget</class>
   <extends>QWidget</extends>
   <header>tachometerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BatteryWidget</class>
   <extends>QWidget</extends>
   <header>batterywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SevenSegmentClock</class>
   <extends>QWidget</extends>
   <header>sevensegmentclock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
